src/Data/JSDate.purs

```diff
@@ -263,3 +272,6 @@ toTimeString dt = runFn2 dateMethod "toTimeString" dt
 -- | Returns the date as a string using the UTC timezone.
 toUTCString :: JSDate -> String
 toUTCString dt = runFn2 dateMethod "toUTCString" dt
+
+-- | Returns the date at a number of milliseconds since 1970-01-01 00:00:00 UTC.
+foreign import fromTime :: Number -> JSDate
```

Comment:
`fromTime` isn't exported or used?

Reply:
It already exists as `fromInstant` too, I'd prefer we use that instead.

Reply:
`fromTime` copes with `NaN` and out of bound `Number`s in the native way, which is not something that makes much sense with `Instant`.

Reply:
Can you add the `fromTime` export? Then I'll finally get this merged in and released.

Reply:
I guess it disappeared again in the convoluted reversion process :wink: